### ISSUE TITLE
Very long variants should be trimmed on page

### ISSUE THREAD

**Issue by pontikos** [phenopolis/phenopolis_genomics_browser#359]
I think very long variant names shoudl be trimmed on page.

For example: 1-12445677-A-CAGTATAGAGTAGAGAGAGAGAGAGGAAG should be displayed as  1-12445677-A-CAGT... 
And you when you mouse over you get the whole variant preview

![image](https://user-images.githubusercontent.com/3852020/120225367-035aad80-c23d-11eb-9958-88c0a53755a7.png)

**Comment by YuanTian1991** [comment]
Hi @pontikos:

I tried to search gene ENSG00000042781 or variant 1-12445677-A-CAGTATAGAGTAGAGAGAGAGAGAGGAAG, but I did not see the long-name variance in the gene page: https://dev-live.phenopolis.org/gene/ENSG00000042781.

And I did not find variant 1-12445677-A-CAGTATAGAGTAGAGAGAGAGAGAGGAAG, how did you get the screenshot?

I am now trying to find a example page with long name to solve the issue, if you get one, you may tell me here.

Best
Tian

**Comment by pontikos** [comment]
This is because we updated to new schema and we now have less variants.
Also @alanwilter i noticed we now have mutlitple times same variant? Probably because DP, FS is different per variant.  Will create separate issue for Alan for that.

@YuanTian1991 if you got to https://dev-live.phenopolis.org/gene/ENSG00000042781 you will find larger variants (indels).  However there are none which are very large anymore.

If you change backend API to prod you should be able to see them as they are in old db.

**Comment by alanwilter** [comment]
@pontikos Indeed, I am aware of the multiple rows for the same variant but it's because of individual_variant table, different patients have different DP etc.
